django/contrib/admin/static/admin/js/inlines.js

```diff
@@ -138,7 +140,25 @@
             if ((maxForms.val() === '') || (maxForms.val() - forms.length) > 0) {
                 addButton.parent().show();
             }
-            // Hide the remove buttons if at min_num.
+
+            const isFormNotEnough = function() {
+                const totalFormNum = ~~totalForms.val();
```

Comment:
Here, we're worried about coverting to an int right? Why not `parseInt()`? (Is that not clearer in intent?)

Reply:
> Here, we're worried about coverting to an int right? Why not `parseInt()`? (Is that not clearer in intent?)

`~~`  could also convert string to int and have high fault tolerance. Of, course, `parseInt()`  and `~~` both fit here and the former is more readable.